web/templates/component/button_error.html.eex

```diff
@@ -0,0 +1 @@
+<%= submit @value, class: "mr3 bn f6 link dim br-pill ph3 pv2 mb2 dib white bg-grey pointer #{@class}", id: @id, type: "button" %>
```

Comment:
agree, I've added the @class for the specific case when we want to hide the "Next" button on the consent page with "display none" or "dn" with Tachyons.